tests/test_module.py

```diff
@@ -1173,7 +1173,17 @@ def test_issue299(self):
             for i in range(100, 1000):
                 r.execute_command('ts.add issue299', i * 10, i)
             actual_result = r.execute_command('ts.range issue299 0 -1 aggregation avg 10')
-            assert actual_result[0] != [0L, '0']       
+            assert actual_result[0] != [0L, '0']
+
+class GlobalConfigTests(ModuleTestCase(REDISTIMESERIES, 
+        module_args=['COMPACTION_POLICY', 'max:1m:1d;min:10s:1h;avg:2h:10d;avg:3d:100d'])):
```

Comment:
indentation?

Reply:
@rafie What would you suggest?

Reply:
sorry, my bad. should really get new glasses.